### ISSUE TITLE
Scroll of Return for new versions

### ISSUE THREAD

**Issue by nickjackson-ff** [TheIllusiveC4/CorpseComplex#44]
It would be wonderful to see the Scroll of Return again in newer versions of Corpse Complex (for Minecraft 1.14x/1.15.x/1.16.x). Its ease of use and unique functionality (consumable item teleport to death location) are still unparalleled, especially as a tool for modpack authors.

Thanks for the consideration!

**Comment by Braasher** [comment]
Yes please add it back, it was such a great and unique item! And no other mod currently has anything similar to it.
